Cargo.toml

```diff
@@ -25,22 +31,22 @@ exclude = [
 ]
 keywords = ["uProtocol", "SDK", "vsomeip", "SOMEIP"]
 license = "Apache-2.0"
-name = "up-client-vsomeip-rust"
 readme = "README.md"
-repository = "https://github.com/eclipse-uprotocol/up-client-vsomeip-rust"
-rust-version = "1.72"
+repository = "https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust"
+rust-version = "1.74"
 version = "0.1.0"
 
-[dependencies]
+[workspace.dependencies]
 async-trait = { version = "0.1" }
 bytes = { version = "1.5" }
 rand = { version = "0.8" }
 regex = { version = "1.10" }
 serde = { version = "1.0", features = ["derive"] }
 serde_json = { version = "1.0" }
-up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", version = "0.1.5" }
+up-rust = { git = "https://github.com/eclipse-uprotocol/up-rust", rev = "3a50104421a801d52e1d9c68979db54c013ce43d" }
```

Comment:
is there a reason why you switched back to the commit instead of e.g. using version `0.1`?

Reply:
Didn't switch. I mentioned on the other PR that I have a separate PR updating to 0.1 over [here](https://github.com/eclipse-uprotocol/up-transport-vsomeip-rust/pull/13).